tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -250,9 +250,11 @@ decap/test_decap.py::test_decap[ttl=pipe, dscp=pipe, vxlan=set_unset]:
 
 decap/test_decap.py::test_decap[ttl=pipe, dscp=uniform, vxlan=disable]:
   skip:
-    reason: "Not supported on backend , broadcom before 202012 release and innovium, x86_64-8111_32eh_o-r0 platform"
+    conditions_logical_operator: or
+    reason: "Not supported on backend, broadcom before 202012 release and innovium, x86_64-8111_32eh_o-r0 platform"
```

Comment:
Add back `Arista 7260CX3 in 202305` in the reason field?

Reply:
Modified and thanks!